work-practices/version-control/source-control.md

```diff
@@ -21,7 +21,8 @@ Our current best practices include guidance for the following aspects of source
 
 Pull requests are the core of a project: they're how changes actually get made, and how work gets pushed live to our clients and their users. Pull requests are important enough that we spend a lot of time thinking about how to do them better, including:
 
-- **When in doubt, seek code review**. At DataMade, we expect nearly all changes to source code to be reviewed by another developer. There are occasional exceptions to this, such as when working on a very tight budget or when addressing an emergency in production, but in general you should expect all your code to be reviewed before merging.
+- **Confirm your build succeeds before requesting review.** If the failure is expected, make note of why in your PR description.
```

Comment:
We weren't using GitHub Actions when this documentation was written. 🙀 Some relevant DataMade history: https://github.com/datamade/how-to/issues/36